mapApp/views/__init__.py

```diff
@@ -1,11 +1,11 @@
-from index import index
-from about import about, contact
-from alerts import alertUsers, postAlertPolygon, readAlertPoint
-from postPoint import postIncident, postNearmiss, postHazard, postTheft, postNewInfrastructure
-from edit import editShape, editHazards, updateHazard
-from termsAndConditions import termsAndConditions
-from disclaimer import disclaimer
-from vis import vis
-from recentReports import recentReports
-from restApi import CollisionList, NearmissList, HazardList, TheftList, FilteredHazardList, FilteredTheftList, OfficialList, UserList, UserDetail, AlertAreaList, AlertAreaDetail, GCMDeviceList, GCMDeviceDetail, APNSDeviceList, APNSDeviceDetail, IncidentList,TinyCollisionList,XHRCollisionInfo,TinyNearMissList,XHRNearMissInfo,TinyHazardList,XHRHazardInfo,TinyTheftList,XHRTheftInfo,TinyNewInfrastructureList,XHRNewInfrastructureInfo
-from pushNotification import pushNotification
+from .index import index
+from .about import about, contact
+from .alerts import alertUsers, postAlertPolygon, readAlertPoint
+from .postPoint import postIncident, postNearmiss, postHazard, postTheft, postNewInfrastructure
+from .edit import editShape, editHazards, updateHazard
+from .termsAndConditions import termsAndConditions
+from .disclaimer import disclaimer
+from .vis import vis
+from .recentReports import recentReports
+from .restApi import CollisionList, NearmissList, HazardList, TheftList, FilteredHazardList, FilteredTheftList, OfficialList, UserList, UserDetail, AlertAreaList, AlertAreaDetail, GCMDeviceList, GCMDeviceDetail, APNSDeviceList, APNSDeviceDetail, IncidentList,TinyCollisionList,XHRCollisionInfo,TinyNearMissList,XHRNearMissInfo,TinyHazardList,XHRHazardInfo,TinyTheftList,XHRTheftInfo,TinyNewInfrastructureList,XHRNewInfrastructureInfo

```

Comment:
I know this wasn't your code, but would you mind cleaning up the formatting here as part of the PR by adding a space after each comma please?